src/System.IO.Compression/src/System.IO.Compression.csproj

```diff
@@ -12,7 +12,7 @@
     <OutputType>Library</OutputType>
     <ProjectGuid>{5471BFE8-8071-466F-838E-5ADAA779E742}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <UseECMAKey Condition="'$(UseECMAKey)'==''">true</UseECMAKey>
```

Comment:
We were not setting this explicitly on unix because the compiler didn't support it when building there so I suspect you will still run into that issue on unix.